### PR TITLE
Add some more sections and details to our docs 📝

### PR DESCRIPTION
**Pull request by bobcatfish**
# Changes

I initially wrote this as part of the initial docs but forgot to commit
it 😅 smile:

tfw you try to update your local checkout and find stashed README
changes that you forgot to submit 🤦‍♀️

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [n/a] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +18,19 @@ The Tekton API enables functionality to be seperated from configuration (e.g. [P
...
+## Want to contribute
+
+Hooray!
+
+- See [CONTRIBUTING.md](CONTRIBUTING.md) for an overview of our processes
+- See [DEVELOPMENT.md](DEVELOPMENT.md) for how to get started
+- Look at our
+  [good first issues](https://github.com/tektoncd/pipeline/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+  and our
+  [help wanted issues](https://github.com/tektoncd/pipeline/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
```
Comment (ncskier) [inline README.md]: Should these links point to the `triggers` repo? (ex: `https://github.com/tektoncd/triggers/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22`) although we don't have issues with those labels right now 😅

Comment (bobcatfish) [reply README.md]: whoops, thanks for catching that! 😅

**Comment by bobcatfish** [comment]
Ready for another look!

**Comment by ncskier** [comment]
Looks great! 🎉 
/lgtm

**Pull request merged**